test/modern_bpf/test_suites/syscall_enter_suite/fsconfig_e.cpp

```diff
@@ -0,0 +1,45 @@
+#include "../../event_class/event_class.h"
+
+#if defined(__NR_fsconfig)
+TEST(SyscallEnter, fsconfigE)
+{
+
+	auto evt_test = get_syscall_event_test(__NR_fsconfig, ENTER_EVENT);
+
+	evt_test->enable_capture();
+
+	/*=============================== TRIGGER SYSCALL ===========================*/
+
+    int fd = 0;
+    uint32_t cmd = 0;
+    const char* key = NULL;
+    const char* value = NULL;
+    int aux = 0;
+	int32_t ret = syscall(__NR_fsconfig, fd, cmd, key, value, aux);
+	assert_syscall_state(SYSCALL_FAILURE, "fsconfig", ret);
+	syscall(__NR_close, fd);
```

Comment:
hmm.. if the syscall fails and `fd` remains `0`, you would close `stdin` here.

Reply:
As above :)